Makefile

```diff
@@ -142,7 +143,12 @@ GOARCH=$(lastword $(subst -, ,$*))
 build/mgmt-%: $(GO_FILES) | bindata lang
 	@echo "Building: $(PROGRAM), os/arch: $*, version: $(SVERSION)..."
 	@# reassigning GOOS and GOARCH to make build command copy/pastable
-	time env GOOS=${GOOS} GOARCH=${GOARCH} go build -i -ldflags "-X main.program=$(PROGRAM) -X main.version=$(SVERSION) ${LDFLAGS}" -o $@ $(BUILD_FLAGS);
+	@# go 1.10 requires specifying the package for ldflags
+	@if go version | grep -qE 'go1.10'; then \
```

Comment:
This is great however can we reverse the test so that we search for 1.9 instead? That way when 1.11 comes out this will still work.

We can ignore tests for 1.8 etc, because they're not supported by the project.